src/workspaces/dashboard/WorkspaceNotifications.tsx

```diff
@@ -0,0 +1,85 @@
+import { InfoBox, Spinner } from '@terra-ui-packages/components';
```

Comment:
Converted to TSX, and the number of lines changed was too large for it to be recognized as a rename.